packages/modules/providers/auth-google/src/services/google.ts

```diff
@@ -85,11 +88,9 @@ export class GoogleAuthService extends AbstractAuthModuleProvider {
       return { success: false, error: "No code provided" }
     }
 
-    const params = `client_id=${this.config_.clientId}&client_secret=${
-      this.config_.clientSecret
-    }&code=${code}&redirect_uri=${encodeURIComponent(
-      this.config_.callbackUrl
-    )}&grant_type=authorization_code`
+    // We can add redirect_uri=${encodeURIComponent(this.config_.callbackUrl)} here as well for enhanced security, although effect is minimal due to having to preconfigure allowed redirect URLs in Google.
```

Comment:
Agh, it seems Google requires it, even though it is optional in the oauth spec. I'll look into implemeting all of this properly now :)